set.mm

```diff
@@ -106076,59 +106340,118 @@ is defined as the Axiom of Choice (first form) of [Enderton] p. 49.  The
+      ZEBIZOZDUULUBZLZCUURUAZBMZAUFACFUGUUTUVKAUUSUVKFUUSUUMUVEEGIZJUDZHIZUVLUU
+      NUEZUVLPZQZNZGUUROZHUHZOZDUULUBZLZCUURUAZUVKUUSUWCCUURUUQCUURUIUUSUULUURK
+      ZUUMUWBUUSUWEUUMNZUVCUULKZUWANZDMZUWHDUJZNZUWBUUSUWFUWIUWJUUSUWFUWIUUSUWE
+      UUMUWFUWILZUUSUWEUUMUUPUWLUUQCUURUQUUPUWFUWIUWFUUPUVBUUOUVAKZNZEUVTOZUWIU
+      WFUUOUULPZUVTKZUULUWPKZUUOUWPKZNZUWOUWFUVMUWPUVPQZNZGUUROZUWQUWFUVMUVLUUL
+      QZNZGUUROZUXCUWEUUMUULUULQZUXFCUKUXEUUMUXGNGUULUURUXDUVMUUMUXDUXGUVLUULJU
+      LUVLUULUULUMRUNUOUXEUXBGUURUXDUXAUVMUXDUVPUUOUVLPUWPUXDUVOUUOUVLUVLUULUUN
+      UPURUVLUULUUOUSUTVAVBVCUVSUXCHUWPUUOUULVDUVNUWPQZUVRUXBGUURUXHUVQUXAUVMUV
+      NUWPUVPUMVEVFVGVHUWRUWSUUOUULCSZVIUUOUULUULUUNVJZVKVLUWNUWTEUWPUVTUVAUWPQ
+      UVBUWRUWMUWSUVAUWPUULTUVAUWPUUOTRUNVMUWHUUPUWONDUUOUXJUVCUUOQZUWGUUPUWAUW
+      OUVCUUOUULVNUXKUVEUWNEUVTUXKUVDUWMUVBUVCUUOUVAVNVEVFRVOVPVQVRWMVSUUSUWHUX
+      KLZDUFUWJUUSUXLDUUSUWGUWAUXKUWAUVAUVTKZUVENZEMUUSUWGNZUXKUVEEUVTVTUXOUXNU
+      XKEUUSUWGUXMUVEUXKUXMUWGUUSUVEUXKLZUXMUWGUUSUXPUXMUVMUVAUVPQZNZGUUROZUWGU
+      USNZUXPLZUVSUXSHUVAESUVNUVAQZUVRUXRGUURUYBUVQUXQUVMUVNUVAUVPUMVEVFVGUXRUY
+      AGUURUXTUVLUURKZUXRUXPUXTUYCUVMUXQUXPUWGUUSUYCUVMUXQUXPLZLLUUSUYCUVMUWGUY
+      DUUSUYCUVMUVOUVLKZUWGUYDLUUQUVMUYELCUVLUURUULUVLQZUUMUVMUUPUYEUULUVLJULUY
+      FUUPUVOUULKUYEUYFUUOUVOUULUULUVLUUNUPZWAUULUVLUVOTWBWCWDUXQUWGUYEUXPUXQUW
+      GUYEUVEUXKUXQUWGUYENZUVENZUVCUVOQZUYFNZUXKUXQUYIUYHUVCUULPUVPQZNUYKUXQUYH
+      UYLUVEUXQUWGUYLUVEWEZUYEUWGUXQUVCUULQWFZUYMUWGUVCUULUVCUULWGWHUYNUYLUVCUV
+      PKZUULUVPKZNZUXQUVEUVCWIKUULWIKUYNUVCUULUDUYLUYQWEDSUXIUVCUULWJUVCUULUVOU
+      VLWIWIWKWLUVEUVDUVBNUXQUYQUVDUVBWNUXQUVDUYOUVBUYPUVAUVPUVCTUVAUVPUULTRWOW
+      PVPWQWRUVCUULUVOUVLUXIWSWTUYFUXKUYJUYFUUOUVOUVCUYGXAXCXBXDXEVRXFXNXGXHXIX
+      JXKXEXOXLXMXPXQUWHDUUOXRVCXSUWBUWHDXTUWKUWADUULYFUWHDYAYBYCXKYDUVJUWDBUVT
+      UVTUUNYEZJYGZYHZUURYHZYIZVUAUYTUURUYRUYSUUNFSYJYKYLASYLYMUVSHVUBUVRUVNVUB
+      KZGUURUYCUVQVUCUVMUYCVUCUVQUVPVUBKZUYCUVPVUAYNZVUDUYCUVOVUAKUVLVUAKVUEUYT
+      VUAUVOUYTUURYOUUNUVLYPYQUVLUURUYTYRUVOUVLVUAYSYTUVPVUAUVOUVLVDUUAVHUVNUVP
+      VUBVNUUBUUCXIUUDUUEUVFUVTQZUVIUWCCUURVUFUVHUWBUUMVUFUVGUWADUULUVEEUVFUVTU
+      UFUUGUUJUUHVOVCUUIUUKXJ $.
+
+    $( Axiom of Choice (first form) of [Enderton] p. 49 corresponds to our
+       Axiom of Choice (in the form of ~ ac3 ).  The proof does not make use of
+       AC, but the Axiom of Regularity is used (by applying ~ dfac2b ).
+       (Contributed by NM, 5-Apr-2004.)  (Revised by Mario Carneiro,
+       26-Jun-2015.)  (Revised by AV, 16-Jun-2022.) $)
     dfac2 $p |- ( CHOICE <-> A. x E. y A. z e. x ( z =/= (/) ->
         E! w e. z E. v e. y ( z e. v /\ w e. v ) ) ) $=
+      ( wac cv c0 wne wel wa wrex wreu wi wral wex wal dfac2b dfac2a impbii ) F
+      CGZHICEJDEJKEBGLDUAMNCAGOBPAQABCDERABCDEST $.
+
+    $( Obsolete proof of ~ dfac2 as of 16-Jun-2022.  Axiom of Choice (first
+       form) of [Enderton] p. 49 implies of our Axiom of Choice (in the form of
```

Comment:
implies of --> implies

Reply:
This typo (and it's not only a typo, but also not semantically correct) was contained in the original comment. I kept it as "proof" that some revision was necessary here...